### ISSUE TITLE
version 0.3.1 returning v0.3.0 with --version

### ISSUE THREAD

**Issue by pahud** [awslabs/fargatecli#90]
```
$ wget https://github.com/jpignata/fargate/releases/download/v0.3.1/fargate-0.3.1-linux-amd64.zip
Saving to: ‘fargate-0.3.1-linux-amd64.zip’

fargate-0.3.1-linux-amd64.zip             100%[===================================================================================>]   9.74M  5.61MB/s    in 1.7s    

2019-05-27 12:49:03 (5.61 MB/s) - ‘fargate-0.3.1-linux-amd64.zip’ saved [10216127/10216127]

$ unzip fargate-0.3.1-linux-amd64.zip 
Archive:  fargate-0.3.1-linux-amd64.zip
  inflating: fargate                 
$ ./fargate --version
fargate version 0.3.0
```

I think just missed to update this file
https://github.com/jpignata/fargate/blob/0a0e05665241b37326bb679436a2b142faa6c37b/cmd/root.go#L20

**Comment by nuttikung** [comment]
Duplicate of #86

**Comment by PaulMaddox** [comment]
Closing duplicate

**Comment by PaulMaddox** [comment]
Fixed in v0.3.2
